src/aig-common/modules/standard/components/ipp-lot-category/lot-category-form/lot-category-form.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
review this file completly. take inspiration from this:
[https://github.com/EneaSys/aig_console/blob/develop/src/aig-common/modules/generic/components/eopoo-type-new-update-form/eopoo-type-new-update-form.component.ts](url)